virtcontainers/sandbox.go

```diff
@@ -740,6 +740,24 @@ func (s *Sandbox) getAndStoreGuestDetails() error {
 	return nil
 }
 
+func rollbackFailingSandboxCreation(ctx context.Context, s *Sandbox) {
+	span, _ := trace(ctx, "rollbackFailingSandboxCreation")
+	defer span.Finish()
+
+	if err := s.destroyCgroups(); err != nil {
+		s.Logger().WithError(err).Error("rollbackFailingSandboxCreation destroyCgroups failed")
+	}
+	if err := s.Release(); err != nil {
```

Comment:
Please release sandbox last in case we want to access any hypervisor or agent resources although we don't right now.

Reply:
corrected, pls check again~